pkgs/development/python-modules/pyutilib/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, nose
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "PyUtilib";
+  name = "${pname}-${version}";
+  version = "5.6.2";
+
+  propagatedBuildInputs = [ nose six ];
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "17rynkkq728b7iy65m2rz9gcycipk0r8xiplfb5lvg5l27zm52gv";
+  };
+
+  doCheck = false;
```

Comment:
Why?

Reply:
because tests are failing:
```
writing manifest file 'PyUtilib.egg-info/SOURCES.txt'
running build_ext
error: [Errno 2] No such file or directory: 'test/config1.out'
builder for '/nix/store/sknnjb4x0fcyszrwpspzy6axjdhlmzkc-python3.6-PyUtilib-5.6.2.drv' failed with exit code 1
error: build of '/nix/store/sknnjb4x0fcyszrwpspzy6axjdhlmzkc-python3.6-PyUtilib-5.6.2.drv' failed
```

Reply:
Then you need to set an appropriate `checkPhase`, something like `test.pyutilib --cat=all`: https://github.com/PyUtilib/pyutilib/blob/master/.travis.yml